.github/workflows/build.yml

```diff
@@ -23,6 +27,9 @@ jobs:
         dotnet-version: 3.1.404
     - name: Setup NUKE
       run: dotnet tool install Nuke.GlobalTool --global
+    - name: Install Workloads
+      # TODO: This is slow. Maybe we can make a docker container with this already done?
+      run: dotnet workload install android android-aot ios maccatalyst maui
```

Comment:
Do we not want to matrix this and have separate CI for core, new Android, old Android, new IOS and old IOS?

Reply:
Also it turns out it's not that slow, it only takes 2 minutes. The runners clearly have very good internet connection.